### PR TITLE
Json/units in columns

### PR DESCRIPTION
**Pull request by thpetsen**

### CODE REVIEW

**Comment by thpetsen** [comment]
Last commit changes order of destinations from csv-input to resulting Table, but since destinations are considered a "set" this should be ok.

> git diff 4dcbdc63495200ffeeecb72c0f3c16876ff93046 434f22d5950a176dbd1a24acb3656e4688632941
diff --git a/examples/pdtable_demo.ipynb b/examples/pdtable_demo.ipynb
index d32be92..0b4a675 100644
--- a/examples/pdtable_demo.ipynb
+++ b/examples/pdtable_demo.ipynb
@@ -594,7 +594,7 @@
      "data": {
       "text/plain": [
        "**farm_animals\n",
-       "other_farm, your_farm, my_farm\n",
+       "your_farm, other_farm, my_farm\n",
        "  species [text]  n_legs [-]  avg_weight [kg]\n",
        "0        chicken         2.0              2.0\n",
        "1            pig         4.0             89.0\n",

File: pdtable/io/parsers/blocks.py [hunk 1]
```diff
@@ -81,7 +82,7 @@ def make_table_json_precursor(cells: CellGrid, **kwargs) -> JsonDataPrecursor:
     # handle multiple columns w. same name
     col_names_raw = cells[2]
     column_names = preprocess_column_names(col_names_raw, fixer)
-    fixer.TableColumNames = column_names
+    fixer.TableColumNames = column_names  # TODO typo... no effect... intended behaviour?
```
Comment (jfcorbett) [inline pdtable/io/parsers/blocks.py:85]: Not originally part of this CR, but apparent bug here:
Typo in "`Colum`" --> this line has no apparent effect
Not clear what the intended behaviour is?

**Review by jfcorbett — APPROVED** [review]

File: pdtable/io/parsers/blocks.py (hunk 1 above)
Comment (jfcorbett) [reply pdtable/io/parsers/blocks.py:85]: Let's complete this PR, and fix this later in its own bug branch.

**Comment by jfcorbett** [comment]
Looks great! Feels much better with the units as part of the columns in JSON.

**Pull request merged**